### ISSUE TITLE
Training on ICDAR 2017 and Inclusion of vertical offset boxes

### ISSUE THREAD

**Issue by aakritigupta0408** [MhLiao/TextBoxes#25]
I am trying to train on the COCO-Text data and my loss is not reducing significantly once it reaches 4.1.
Also, How is the inclusion of vertical offset default boxes being done in the code provided. 

Regards,
Aakriti

**Comment by MhLiao** [comment]
In general, more default boxes is better. You can try to add the vertical offset default boxes. COCO-Text is much challenging, so more tricks are needed.

**Comment by MhLiao** [comment]
@aakritigupta0408  The annotation of COCO-Text is too dirty, which cause the problem of training. You can try to set the train_on_difficult_gt as False.

**Comment by aakritigupta0408** [comment]
@MhLiao Can you please elaborate on the calculation of num_priors as I can calculate them to be 11 and 12 instead of the number mentioned by you in the code.

**Comment by aakritigupta0408** [comment]
Thanks a lot for the prompt reply!
While training on the Synthetic Dataset do I have to lead any pre-trained weights or train from scratch?
Also what is the batch size for training and how is the learning rate changing during the training

**Comment by MhLiao** [comment]
@aakritigupta0408 Training details can be found in the paper.
